wdio/pages/applications.js

```diff
@@ -248,6 +259,78 @@ class Application extends SalesForce {
         //Switch to Accessibility Title Iframe
         await browser.switchToParentFrame();
     }
+
+    async addCreditApprovalConditions(approvalCondition = 'Demo approval condition', conditionIndex=1){
+        await browser.pause(10000);
```

Comment:
It's better to avoid static waits. It will be better if the waits of the elements can be handled with waitFor... methods or reloadIfNot... methods. As that needs to be applied in goToApplicationTabWithText method()